webpack.config.js

```diff
@@ -0,0 +1,85 @@
+/* Thanks to Ashwin Hariharan
+https://medium.freecodecamp.com/webpack-for-the-fast-and-the-furious-bf8d3746adbd#.4ijclp4kt */
+
+var webpack = require("webpack"),
+    path = require("path"),
+    config = require("src/config.js");
+
+var lib_dir = __dirname + "/src/static/libs",
+    node_dir = __dirname + "/node_modules";
+
+var config = {
+    resolve: {
+        alias: {
+            bootstrap: node_dir + "/bootstrap/dist/js/bootstrap.min.js",
```

Comment:
webpack looks in `node_modules` as default behavior... so these `node_dir` aliases seem redundant